Plugins/Mute/Commands/MuteCommand.cs

```diff
@@ -0,0 +1,41 @@
+﻿using Data.Models.Client;
+using SharedLibraryCore;
+using SharedLibraryCore.Commands;
+using SharedLibraryCore.Configuration;
+using SharedLibraryCore.Interfaces;
+
+namespace Mute.Commands;
+
+public class MuteCommand : Command
+{
+    public MuteCommand(CommandConfiguration config, ITranslationLookup translationLookup) : base(config,
+        translationLookup)
+    {
+        Name = "mute";
+        Description = translationLookup["PLUGINS_MUTE_COMMANDS_MUTE_DESC"];
+        Alias = "mu";
+        Permission = EFClient.Permission.Moderator;
+        RequiresTarget = true;
+        Arguments = new[]
+        {
+            new CommandArgument
+            {
+                Name = translationLookup["COMMANDS_ARGS_PLAYER"],
+                Required = true
+            }
+        };
+    }
+
+    public override async Task ExecuteAsync(GameEvent gameEvent)
+    {
+        var muteManager = new MuteManager();
```

Comment:
Don't new this up every time the command is executed. Just initialize it once in the constructor as a private class field. Commands are singletons, ie every time the ExecuteAsync is called it's on the same command instance